babel 2.11.0

skupr-anaconda opened the pull request:
**Jira ticket:** [PKG-934](https://anaconda.atlassian.net/browse/PKG-934) (babel 2.11.0)

**The upstream data:**
Github releases:  https://github.com/python-babel/babel/releases
[Diff between the latest and previous upstream releases](https://github.com/python-babel/babel/compare/v2.9.1...v2.11.0)
License: INFO: https://github.com/python-babel/babel/blob/master/LICENSE
Changelog: https://github.com/python-babel/babel/blob/master/CHANGES.rst
Requirements:
 * setup.py:  https://github.com/python-babel/babel/blob/v2.11.0/setup.py

**_Actions:_**

- Skip py<37 
- Fix python in host and run
- Add pip check
- Fix home url
- Add license_family
- Fix license name 
- Fix doc_url

**Notes**

- `jupyterlab_server 2.16.5` requires it https://github.com/AnacondaRecipes/jupyterlab_server-feedstock/pull/20

**Additional info**
<details>

**Package's statistics**
<details>

 * Priority C | effort: easy | Category: anaconda | subcategory: dependency | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-s390x', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 2.11.0
 * Release on PyPi:
    * version: 2.11.0
    * date: 2022-11-01T10:46:06
* [PyPi history](https://pypi.org/project/babel/#history)
 * Popularity: 
    * 3 months downloads: 598592.0
    * All time:  8241813 downloads -  babel  2.11.0  Utilities to internationalize and localize Python applications  

</details>

**Other checks:**

<details>

1. - [x] https://github.com/python-babel/babel/tree/v2.11.0 exists
3. - [x] Check the pinnings
4. - [x] https://github.com/python-babel/babel/blob/master/CHANGES.rst exists
5. - [x] Additional research
    https://github.com/python-babel/babel/issues
    200
6. - [x] `dev_url` is present
    https://github.com/python-babel/babel
7. - [x] `doc_source_url` is present
    https://github.com/python-babel/babel/blob/master/docs/index.rst
8. - [x] Verify that the `build_number` is correct
9. - [x] has `setuptools`
10. - [x] Verify if the package needs `wheel`
11. - [x] `pip` in test
12. - [x] Verify the test section
13. - [x] Verify if the package is `architecture specific`
14. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
15.  - [x] license_file: LICENSE is present
16. - [x] License: BSD
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier                           |
|:-------------------------------------|
| 0BSD                                 |
| BSD-1-Clause                         |
| BSD-2-Clause                         |
| BSD-2-Clause-Patent                  |
| BSD-2-Clause-Views                   |
| BSD-3-Clause                         |
| BSD-3-Clause-Attribution             |
| BSD-3-Clause-Clear                   |
| BSD-3-Clause-LBNL                    |
| BSD-3-Clause-Modification            |
| BSD-3-Clause-No-Military-License     |
| BSD-3-Clause-No-Nuclear-License      |
| BSD-3-Clause-No-Nuclear-License-2014 |
| BSD-3-Clause-No-Nuclear-Warranty     |
| BSD-3-Clause-Open-MPI                |
| BSD-4-Clause                         |
| BSD-4-Clause-Shortened               |
| BSD-4-Clause-UC                      |
| BSD-Protection                       |
| BSD-Source-Code                      |
| FreeBSD-DOC                          |

17. - [ ] License family is NOT present
    
</details>

**Check depdendency issues:**

<details>
../aggregate/babel-feedstock/recipe/meta.yaml
Dependencies: ['pip', 'pytz >=2015.7', 'python >=3.6', 'pytz', 'setuptools']


noarch:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

linux-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

linux-ppc64le:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

linux-s390x:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

osx-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

osx-arm64:
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

win-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

</details>


**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/babel-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/babel-feedstock/tree/2.11.0)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/babel)
* [Diff between upstream and feature branch](https://github.com/conda-forge/babel-feedstock/compare/main...AnacondaRecipes:2.11.0)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/babel-feedstock/compare/master...AnacondaRecipes:2.11.0)
* [The last merged PRs](https://github.com/AnacondaRecipes/babel-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>

**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 2.11.0 git@github.com:AnacondaRecipes/babel-feedstock.git
```

skupr-anaconda committed:
DOR update to 2.11.0

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:19: avoid_noarch: noarch: python packages should be avoided
ERROR: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:26: missing_wheel: For pypi packages, wheel should be present in the host section
ERROR: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:38: missing_pip_check: For pypi packages, pip check should be present in the test commands
ERROR: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:42: incorrect_license: The recipe's `about/license` key is not an SPDX compliant license or license exception, closest match: BSD-3-Clause
WARNING: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:51: missing_license_url: The recipe is missing a license_url
ERROR: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:51: missing_license_family: The recipe is missing the `about/license_family` key.
WARNING: /tmp/abs_beinfosrbi/clone/recipe/meta.yaml:51: http_url: http://babel.pocoo.org/ is not https
Errors were found

</details>

skupr-anaconda committed:
Update to 2.11.0

- Skip py<37

- Fix python in host and run

- Add pip check

- Fix home url

- Add license_family

- Fix license name

- Fix doc_url

[PKG-934]

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_ecfz_asnte/clone/recipe/meta.yaml:24: missing_wheel: For pypi packages, wheel should be present in the host section
WARNING: /tmp/abs_ecfz_asnte/clone/recipe/meta.yaml:52: missing_license_url: The recipe is missing a license_url
Errors were found

</details>

skupr-anaconda committed:
Fix pip check

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_9delr23maw/clone/recipe/meta.yaml:24: missing_wheel: For pypi packages, wheel should be present in the host section
WARNING: /tmp/abs_9delr23maw/clone/recipe/meta.yaml:52: missing_license_url: The recipe is missing a license_url
Errors were found

</details>

skupr-anaconda committed:
Add wheel to host

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_72pgnn6vdp/clone/recipe/meta.yaml:53: missing_license_url: The recipe is missing a license_url
Warnings were found

</details>

ryanskeith reviewed (approved):
LGTM

ELundby45 reviewed (approved):
LGTM

varlackc reviewed (approved):
LGTM

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_c8cbqqm389/clone/recipe/meta.yaml:53: missing_license_url: The recipe is missing a license_url
Warnings were found

</details>
